runtime/compiler/runtime/J9CodeCacheManager.cpp

```diff
@@ -723,3 +723,9 @@ J9::CodeCacheManager::printOccupancyStats()
       }
    }
 
+
+intptrj_t
```

Comment:
Provide a version of `findHelperTrampoline` that returns an `intptrj_t` type rather than `void *`.  This eliminates a number of casts that would be immediately required when using this function.